spec/spec_helper.rb

```diff
@@ -2,6 +2,9 @@
 ENV["RAILS_ENV"] ||= 'test'
 
 require 'simplecov'
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.